### PR TITLE
[tests-only][full-ci] added test for mail notification if space admin delete access to space

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR forward ports https://github.com/owncloud/ocis/pull/6657

#### test environment:
* locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:  if CI is happy

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 1]
```diff
@@ -215,6 +215,10 @@ public function getSpaceByName(string $user, string $spaceName): array {
 	 */
 	public function getSpaceIdByName(string $user, string $spaceName): string {
 		$space = $this->getSpaceByName($user, $spaceName);
+		Assert::assertIsArray($space, "Space with name $spaceName not found");
+		if (!isset($space["id"])) {
+			throw new Exception(__METHOD__ . " space with name $spaceName not found in $space");
+		}
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/acceptance/features/bootstrap/SpacesContext.php:221]: Yeah, true. I think this line can be removed
```
Assert::assertIsArray($space, "Space with name $spaceName not found");
```

[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/acceptance/features/bootstrap/SpacesContext.php:221]: this is checking the space id property, so I think good to have it there
```php
if (!isset($space["id"])) {
```

File: tests/acceptance/features/bootstrap/NotificationContext.php [hunk 2]
```diff
@@ -239,8 +239,8 @@ public function userShouldHaveReceivedTheFollowingEmailFromUserAboutTheShareOfPr
 				[
 					"code" => "%space_id%",
 					"function" =>
-						[$this->spacesContext, "getSpaceIdByNameFromResponse"],
-					"parameter" => [$spaceName]
+						[$this->spacesContext, "getSpaceIdByName"],
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/NotificationContext.php:242]: `getSpaceIdByName` was already there is master. :+1:

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by v-scharf — APPROVED** [review]

**Pull request merged**